data/json/mapgen/steel_mill/steel_mill_z1.json

```diff
@@ -0,0 +1,141 @@
+        "v....]l   l#   #         #     h]]]]]]]]]]]----_______________________.v",
+        "v....]l   l#   #+#+#+#+#+#     h]------------.._______________________.v",
+        "v....]l    +   #t#t#t#t#t#     h]------------.._______________________.v",
+        "v....]l   l#   ###########      *--___________________________,,,,,,,,.v",
+        "v....]l   l#             +      *--___________________________________.v",
+        "v....]]]]]]]D            +      ]--___________________________________.v",
+        "v..........]]]]**]]]]**]]]++##++]--,,,,,,,,___________________________.v",
+        "v........................]      ]--___________________________,,,,,,,,.v",
+        "v........................*      *--___________________________________.v",
+        "v....]]]]]]]]]]]]]]].....*      *--___________________________________.v",
+        "v....]MMMM        <].....]      ]--,,,,,,,,___________________________.v",
+        "v....]FFFFE        ]]]]]]]++##++]----------___________________,,,,,,,,.v",
+        "v....]F  x||||     ]lllll]      ]..............____________............v",
+        "v....]F  x||||     R     +      ]]]]]]]]]]]]]]]____________vvvvvvvvvvvvv",
+        "v....]FFFF  ||     ]+]]]]]]]]]++#CCCCCCCCCCCCC]____________............v",
+        "v....]MMMM  ||        DD  DD ]  #  C  C  C   C]____________.)))))))))).v",
+        "v....]FFFF  ||        DD  DD ]  #  C  C  C   C]____________.) b  bbb ).v",
+        "v....]F  x||||        DD  DD ]  #  C  C  C   C]____________-+       b).v",
+        "v....]F  x||||        DD  DD ]  +            C]____________.) bbb  bb).v",
+        "v....]FFFF  ||               +  ##############]____________.)b  b  bb).v",
+        "v....]MMMM  ||               +  #CCCCCC#sssss]]____________.)))))))))).v",
+        "v....]FFFF  ||               ]  #C     #s s s]]____________............v",
+        "v....]F  x||||||||||         ]  #      #s   ]]'____________............v",
+        "v....]F  x||||||||||        <]  +      +   ]]''____________............v",
+        "v....]FFFFE                 <]  #      #ss]]   ____________............v",
+        "v....]P  P                ee@]++]]]]]]]]]]] b ,____________............v",
+        "v....]P  P  <<  mc m      eee]   hTh hTh f]b   ____________''''''''''''v",
+        "v....]MMMMPPPPPPPME       eee]           C]  b,____________''OOOOOOO'''v",
+        "v....]MEEMPPPPPPPEM          ]    DV     0]    ____________''xoooooO'''v",
+        "v....]mccm  <<  mPPm         ]++]]]]]]]]]]]   ,____________''xoooooO'''v",
+        "v....]           PP          ]           ;]    ____________''OOOOOOO'''v",
+        "v....]#####     <PP<         x            x   ,____________''''''''''''v",
+        "v....]<<A R     <PP<         x            x    ____________''OOOOOOO'''v",
+        "v....]    R      PP          x            x   ,____________''xoooooO'''v",
+        "v....]l   #     mPPm         x            x  b ____________''xoooooO'''vvvvvvvvvvvvvvvvvvvvvvvvv",
```

Comment:
No fence here, is that intentional?

Reply:
As far as I can see, all of those lines have fences? Regardless, I'm in the process of updating the map; I'll be editing the fencing a bit, so I'll try to catch this. Thanks for pointing it out!